src/router/index.js

```diff
@@ -244,6 +250,14 @@ const routes = [
     props: (route) => ({ code: route.query.code }),
     meta: { pageTitle: 'Clever Authentication' },
   },
+  {
+    path: '/auth-classlink',
```

Comment:
Can I suggest that this path be edited to `/classlink-auth` for consistency?

Reply:
I think auth-classlink is consistent with auth-clever and auth-email-link